### ISSUE TITLE
Binder asRequired working only if ComboBox has items

### ISSUE THREAD

**Issue by nemojmenervirat** [vaadin/flow#5082]
- **Description of the bug**
In latest version I get (TypeError) : Cannot read property 'enableClientValidation' of undefined when using binder as required with combobox.

- **Minimal reproducible example**
```
@Route
public class MainView extends VerticalLayout {

	class TestBean {
		private String desc;

		public String getDesc() {
			return desc;
		}

		public void setDesc(String desc) {
			this.desc = desc;
		}

	}

	public MainView() {
		ComboBox<String> comboBox = new ComboBox<>();

		// works if setItems is before binding
		// comboBox.setItems(Arrays.asList("item1", "item2"));

		Binder<TestBean> binder = new Binder<>();
		binder.forField(comboBox).asRequired().bind(TestBean::getDesc, TestBean::setDesc);

		// does not work if setItems is after binding
		// comboBox.setItems(Arrays.asList("item1", "item2"));

		add(comboBox);
	}

}
```

- **Expected behavior**
Binding with ComboBox works without errors when setItems is called after binding

- **Actual behavior**
![image](https://user-images.githubusercontent.com/3105705/52940790-a9819280-3367-11e9-9757-9828ae6c9094.png)

- **Versions**:
    - Flow version: 12.0.6
    - Java version: 1.8
    - Browser version: latest Chrome and Firefox

**Comment by ZheSun88** [comment]
the client-side error shows in 12.0.7 too.

**Comment by Lutz-W** [comment]
I got the same Problem with 13.0.0.

I wrote a custom validator, which calls `.setRequiredIndicatorVisible(true)` if another component holds a value. The first time I call `binder.readBean()` there's no error message but after the second call the error is shown. 

The other component also has a valueChangeListener, which enables/disables the component in question, if its own value is present. I think the error is shown to the user because the component is disabled and gets an update to the visibility of the required indicator.

Also, I get the following messages inside my console:

> 2019-03-14 15:33:31.841  WARN 4540 --- [nio-8080-exec-1] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'opened'. Request is ignored.
2019-03-14 15:33:31.843  WARN 4540 --- [nio-8080-exec-1] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'value'. Request is ignored.
2019-03-14 15:33:31.843  WARN 4540 --- [nio-8080-exec-1] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'filter'. Request is ignored.
2019-03-14 15:33:31.843  WARN 4540 --- [nio-8080-exec-1] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'opened'. Request is ignored.
2019-03-14 15:33:35.519  WARN 4540 --- [nio-8080-exec-2] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'opened'. Request is ignored.
2019-03-14 15:33:35.520  WARN 4540 --- [nio-8080-exec-2] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'value'. Request is ignored.
2019-03-14 15:33:35.520  WARN 4540 --- [nio-8080-exec-2] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'filter'. Request is ignored.
2019-03-14 15:33:35.520  WARN 4540 --- [nio-8080-exec-2] c.v.f.s.c.rpc.MapSyncRpcHandler          : Property update request for disabled element is received from the client side. The property is 'opened'. Request is ignored.
